<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTimeLineEdit</class>
 <widget class="QWidget" name="CTimeLineEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>133</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="ViewCombo">
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Tempo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="TempoSpin"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="UpperSpin"/>
   </item>
   <item>
    <widget class="QComboBox" name="LowerCombo">
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
